mlflow/server/js/src/model-registry/components/SchemaTable.tsx

```diff
@@ -54,16 +54,44 @@ export class SchemaTableImpl extends React.PureComponent<Props> {
     );
   };
 
-  getSchemaTypeRepr = (schemaTypeSpec: any) => {
-    let { type } = schemaTypeSpec;
-    if (schemaTypeSpec.type === MODEL_SCHEMA_TENSOR_TYPE) {
-      type = `Tensor (dtype: ${schemaTypeSpec['tensor-spec'].dtype}, shape: [${schemaTypeSpec['tensor-spec'].shape}])`;
+  getSchemaTypeRepr = (schemaTypeSpec: ColumnSpec | TensorSpec): string => {
```

Comment:
Rendering the cell value as code might help improve readability:

```
array{
    str?: string,
    num?: integer,
    obj: array({
        bool: boolean
    })
})
```

Reply:
Yeah I agree, I can make the change.

Also what do you think about reducing the width of the code sample section? I'm guessing people mostly copy and paste it anyway, but I'm not sure:

<img width="1271" alt="Screenshot 2023-10-24 at 2 47 02 PM" src="https://github.com/mlflow/mlflow/assets/148037680/b665deec-1df6-47d1-9d11-be6389347e1e">

Reply:
Sounds good to me. It looks too wide. We can wrap the long lines if necessary.